### PR TITLE
more semantic token colors

### PR DESCRIPTION
**Pull request by xiyaowong**

### CODE REVIEW

File: themes/OneNord-color-theme.json [hunk 1]
```diff
@@ -1468,6 +1468,10 @@
       "regexp": "#88C0D0",
       "operator": "#B589D3",
       "decorator": "#81A1C1",
-      "module": "#C0D0E0"
+      "module": "#C0D0E0",
+      "property": "#81A1C1",
+      "struct": "#EBCB8B",
+      "class": "#EBCB8B",
```
Comment (s1e2b3i4) [inline themes/OneNord-color-theme.json]:
```suggestion
```

**Review by s1e2b3i4 — CHANGES_REQUESTED** [review]
Hey @xiyaowong, thank you for your contribution!
The only thing I would like to change is `"class": "#EBCB8B"`. The reason for that is, that I would like to have the declaration of a class colored differently than the instantiation of it. Therefore, I already set `"class.declaration": "#EBCB8B"`.
What one could do is set the `class` to the color of a function call. But this would then color all class occurrences in that color, and not only the instantiations. And hence I try to be in line with the original theme, this would look not similar anymore.
Besides that, everything looks great and I would merge the PR.

**Comment by xiyaowong** [comment]
Done~

**Review by s1e2b3i4 — APPROVED** [review]

**Comment by s1e2b3i4** [comment]
Thanks for your understanding and your contribution!

**Pull request closed without merging**